docs/v3/status-socket.rst

```diff
@@ -0,0 +1,53 @@
+Status Socket
+=============
+
+Basic information about running OnionBalance can be obtained by querying
+so called status socket.
+
+Status socket is a Unix socket file created by OnionBalance. It is
+automatically closed by OnionBalance after reading it to the end.
+
+Example:
+
+.. code-block::
+
+    socat - unix-connect:/var/run/onionbalance/control | json_pp -json_opt pretty
+    {
+       "services" : [
+          {
+             "instances" : [
+                {
+                   "introModified" : "2020-06-16 19:59:28",
```

Comment:
We should also have the received instance descriptor timestamp here. There is a `received_ts` for that so it shouldn't be hard.

Reply:
Done